### PR TITLE
Feature/ados123 35 layout more screen

### PR DESCRIPTION
**Pull request by marcos-wms**

### CODE REVIEW

File: src/app/screens/More/styles.js [hunk 1]
```diff
@@ -3,11 +3,34 @@ import styled from 'styled-components';
...
+
+export const ContainerAccordionItems = styled.View`
+  width: 100%;
 `;
 
 export const TextWhite = styled.Text`
   color: ${(props) => props.theme.colors.white};
   margin-vertical: 20;
   text-align: center;
+  font-family: 'SF-Pro-Display-Bold';
```
Comment (jhonwms) [inline src/app/screens/More/styles.js:21]: usar la del theme

File: src/app/components/common/MoreAccordionItem/styles.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+`;
+
+export const TextContent = styled.Text`
+  color: ${(props) => props.theme.colors.white};
+  text-align: left;
+  padding-right: 40;
+  width: auto;
+  margin-vertical: 20;
+  font-size: 15;
+  font-family: 'SF-Pro-Display-Light';
```
Comment (jhonwms) [inline src/app/components/common/MoreAccordionItem/styles.js:36]: fuentes del theme

File: src/app/screens/More/styles.js [hunk 3]
```diff
@@ -3,11 +3,34 @@ import styled from 'styled-components';
 export const Container = styled.View`
   flex: 1;
   margin-top: 20%;
-  justify-content: space-between;
+  justify-content: flex-start;
+`;
+
+export const ContainerListItems = styled.View`
```
Comment (usaqui-albert) [inline src/app/screens/More/styles.js:9]: Why are you using the same styles in two different styled components? You can have only one e.g: `FullWidthContainer`

File: src/app/screens/More/index.js [hunk 4]
```diff
@@ -1,33 +1,111 @@
...
+                  onPressNavigate={onPressNavigate}
+                  folding={up}
+                />
+              )}
             />
-          )}
-        />
-      </Container>
+          </ContainerAccordionItems>
+          <ListButton onPress={() => logout()} underlayColor={theme.colors.blueUnderlay}>
```
Comment (usaqui-albert) [inline src/app/screens/More/index.js:104]: `<ListButton onPress={logout} ...` is shorter.

File: src/app/screens/More/index.js [hunk 5]
```diff
@@ -1,33 +1,111 @@
...
 import MoreListItem from '@common/MoreListItem';
-import { list } from './mock';
-import { Container, TextWhite } from './styles';
+import { theme } from 'app/themes';
+import MoreAccordionItem from '@common/MoreAccordionItem';
+import up from '@images/up.png';
+import dropDown from '@images/drop-down.png';
+import {
+  list, accountList, aboutUsList, needSomeHelpList,
+} from './mock';
```
Comment (usaqui-albert) [inline src/app/screens/More/index.js:13]: This are not "mock" lists, `navigationLabels.js` is a better name.

File: src/app/components/common/MoreAccordionItem/index.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  <ContainerAccordion>
+    <TouchableHighlight onPress={onPress} underlayColor={theme.colors.blueUnderlay}>
+      <ItemAccordion>
+        <ContainerTitle>
+          <TextWhite numberOfLines={3}>{label}</TextWhite>
+        </ContainerTitle>
+        <DropDown resizeMode="contain" source={element ? dropDown : folding} />
+      </ItemAccordion>
+    </TouchableHighlight>
+    { element
```
Comment (usaqui-albert) [inline src/app/components/common/MoreAccordionItem/index.js:37]: I think a more clean indentation is the following:
```
{element && (
    <FlatList
        data={list}
        ....
    />
)}
```

**Pull request merged**